tests/slack_notifier/utils.py

```diff
@@ -0,0 +1,27 @@
+import logging
+import pycurl
+
+class SlackTestHandler(logging.StreamHandler):
+    
+    '''Custom handler inheriting from StreamHandler to retain the
+    formatter functionality.
+    Overriding the emit function to use pycurl to send messages to slack
+    (as it turned out to be the most robust option).
+    '''
+    c = pycurl.Curl()
```

Comment:
If you are not using it, just do not import it (`pep8`) unnecessary, confusing part of the code that only consumes resources. Even if you want preserve similarities, than inheritance is your friend, not repeating the code.